curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-regular-expressions-by-building-a-spam-filter/641cd18eb67c661d8a9e11f3.md

```diff
@@ -32,7 +32,7 @@ assert.match(code, /const\s+result\s*=/);
 Your `result` variable should have the value of `document.getElementById('result')`.
 
 ```js
-assert.deepEqual(result, document.getElementById('result'));
+assert.match(code, /const\s+result\s*=\s*document\.getElementById\(\s*('|")result\1\s*\)/);
```

Comment:
What was wrong with this one? Is it that there's other ways to get the element?

Reply:
Test passes with seed code, since `result` gets defined automatically.